tests/test_incremental.py

```
@@ -0,0 +1,214 @@
+            "0\t0\trep:CLUS_001\n"
+            "0\t1\trep:CLUS_002\n"
+            "0\t2\tid=mzspec::file.raw:scan:123:PEPTIDE/2\n"
+        )
+        titles_path = str(tmp_path / 'test.scan_titles.txt')
+        Path(titles_path).write_text(titles_content)
+
+        df = parse_all_scan_titles([titles_path])
+
+        assert len(df) == 3
+        # Check rep identification
+        assert df.iloc[0]['is_rep'] == True
```

Ruff (0.15.10)

[error] Avoid equality comparisons to `True`; use `df.iloc[0]['is_rep']:` for truth checks

Replace with `df.iloc[0]['is_rep']`

(E712)